ext/node/ops/fs.rs

```diff
@@ -185,16 +185,24 @@ where
     let mut bytes_per_sector = 0;
     let mut available_clusters = 0;
     let mut total_clusters = 0;
-    // SAFETY: Normal GetDiskFreeSpaceW usage.
-    let code = unsafe {
-      GetDiskFreeSpaceW(
-        root.as_ptr(),
-        &mut sectors_per_cluster,
-        &mut bytes_per_sector,
-        &mut available_clusters,
-        &mut total_clusters,
-      )
-    };
+    let mut code = 0;
+    let mut retries = 0;
+    while code == 0 && retries < 10 {
+      if retries > 0 {
+        std::thread::sleep(std::time::Duration::from_millis(1));
+      }
+      // SAFETY: Normal GetDiskFreeSpaceW usage.
+      code = unsafe {
+        GetDiskFreeSpaceW(
+          root.as_ptr(),
+          &mut sectors_per_cluster,
+          &mut bytes_per_sector,
+          &mut available_clusters,
+          &mut total_clusters,
+        )
+      };
+      retries += 1;
+    }
```

Comment:
Libuv seems to not retry on a certain error. Maybe we should do the same? https://github.com/libuv/libuv/blob/fa6745b4f26470dae5ee4fcbb1ee082f780277e0/src/win/fs.c#L2719

Reply:
In our case the canonicalize call above will catch the normal error cases, so imo it's okay if we don't filter the error or have a slightly longer timeout